lib/jira/client/axios.js

```diff
@@ -1,86 +1,61 @@
 const axios = require('axios')
 const jwt = require('atlassian-jwt')
+const querystring = require('querystring')
 const url = require('url')
 
 const { logger } = require('probot/lib/logger')
 
 const instance = process.env.INSTANCE_NAME
 const iss = 'com.github.integration' + (instance ? `.${instance}` : '')
 
+/*
+ * Create an Atlassian request from an Axios config.
+ * Compatible with jwt.createQueryStringHash.
+ */
+jwt.reqFromAxiosConfig = (config) => {
+  const parsedUrl = url.parse(config.url)
+  const parsedQuery = querystring.parse(parsedUrl.query)
+
+  return {
+    method: config.method,
+    originalUrl: parsedUrl.pathname,
+    query: Object.assign(parsedQuery, config.params)
```

Comment:
I think we were passing `parsedUrl.query` directly in this object for the `query` field when constructing the token before this change. Might be missing context, but is there any reason to change the payload here for the Axios upgrade? Not sure if this could break the token verification if `config.params` adds some extra fields to the `query` which the verification doesn't account for.

I love the idea of breaking this out into a method but maybe we can keep the parameters for the payload the same as before?